dissect/target/plugins/os/unix/linux/fortios/_os.py

```diff
@@ -2,56 +2,97 @@
 
 import gzip
 import io
+from base64 import b64decode
 from binascii import crc32
+from datetime import datetime
+from tarfile import ReadError
 from typing import BinaryIO, Iterator, Optional, TextIO
 
-from dissect.util import cpio
+from Crypto.Cipher import AES
+from dissect.util import cpio, ts
 from dissect.util.stream import OverlayStream
 
 from dissect.target.filesystem import Filesystem
 from dissect.target.filesystems.tar import TarFilesystem
-from dissect.target.helpers.record import UnixUserRecord
+from dissect.target.helpers.fsutil import open_decompress
+from dissect.target.helpers.record import TargetRecordDescriptor
 from dissect.target.plugin import OperatingSystem, export
 from dissect.target.plugins.os.unix.linux._os import LinuxPlugin
 from dissect.target.target import Target
 
+FortiOSUserRecord = TargetRecordDescriptor(
+    "fortios/user",
+    [
+        ("string", "name"),
+        ("string", "groups"),
+        ("string", "password"),
+        ("path", "home"),
+    ],
+)
+
+
+class FortiOSPlugin(LinuxPlugin):
+    """FortiOS Dissect Plugin"""
```

Comment:
```suggestion
    """FortiOS plugin for various Fortinet appliances."""
```
I tried to find another OS plugin with a docstring but they're pretty rare 😄 a small change like this is probably fine.